media/css/firefox/partners/desktop.less

```diff
@@ -1,861 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-@import '../../sandstone/lib.less';
-
-/* {{{ Journal font */
-
-@font-face {
-    font-family: 'Journal';
-    src: url('/media/fonts/journal-webfont.eot');
-    src: url('/media/fonts/journal-webfont.eot?#iefix') format('embedded-opentype'),
-         url('/media/fonts/journal-webfont.woff') format('woff'),
-         url('/media/fonts/journal-webfont.ttf') format('truetype');
```

Comment:
Good spot. I seem to remember this particular font is not very good for l10n, so I say let's remove :smile: